tests/installation/welcome.pm

```diff
@@ -192,6 +192,11 @@ sub run {
             send_key 'alt-' . $hotkey{$product};
         }
         else {
+            if (check_var('VIDEOMODE', 'ssh-x') && is_ppc64le) {
+                # on ssh-X we need to wait for some popups to go away on ppc64le
+                record_soft_failure('poo#110470: X-Forwarding via SSH is very slow on ppc64le');
```

Comment:
This unfortunately records a soft failure without checking for the actual error condition. Also I missed that you wanted to point to a progress ticket, not a bug in bugzilla, which should be avoided. Please only reference product issues with soft failure